osgp/shared/osgp-dto/src/main/java/org/opensmartgridplatform/dto/valueobjects/smartmetering/ConfigurationFlagsDto.java

```diff
@@ -17,18 +17,18 @@
 
     private static final long serialVersionUID = 8360475869038077578L;
 
-    private List<ConfigurationFlagDto> configurationFlag;
+    private final List<ConfigurationFlagDto> flags;
 
-    public ConfigurationFlagsDto(final List<ConfigurationFlagDto> configurationFlag) {
-        this.configurationFlag = new ArrayList<>(configurationFlag);
+    public ConfigurationFlagsDto(final List<ConfigurationFlagDto> flags) {
+        this.flags = new ArrayList<>(flags);
     }
 
     @Override
     public String toString() {
-        return this.configurationFlag == null ? "Flags[none]" : String.format("Flags%s", this.configurationFlag);
+        return String.format("Flags %s", this.flags);
     }
 
-    public List<ConfigurationFlagDto> getConfigurationFlag() {
-        return new ArrayList<>(this.configurationFlag);
+    public List<ConfigurationFlagDto> getFlags() {
+        return new ArrayList<>(this.flags);
```

Comment:
consider returning a Collections.unmodifiableList(list);

Reply:
I normally don't bother hiding internal collection fields like this because it causes more suffering than happiness IMO (in support, JAXB doesn't do it either). In this case the code was already like this and I decided not to change it to avoid problems. Some calling code might modify the collection after getting it, which would result in bugs when the implementation is changed (either by exposing the list or by returning an unmodifiable copy).

Therefore I decided to leave it as it is.